src/engine/evaluate.ts

```diff
@@ -30,3 +45,7 @@ export const evaluateCondition = (
     return matches(value);
   });
 };
+
+export const testables = {
```

Comment:
I do not like adding test code into source.

Reply:
Would you rather have the internal filterPageViews exported?
I'm ok with too.
I think exporting the internals inside the `testable` context gives a hint to the developer that it's methods are not to be used out of testing context as the exported ones.

Reply:
Ahh sorry I did not notice the original was not exported.

Then on this my view would be as follows, internal functions to modules should not be tested, just what is exported... again I know this is just an opinion, and many good points why doing it another way is correct.

I just find this way forces you to split modules at the same level as you would want to test them.

Due to time constraints - I am not bothered what we do now. Going forward we should agree to make sure stuff is consistent I think that is even more important.